data/gui/widget/spinner_default.cfg

```diff
@@ -0,0 +1,220 @@
+#textdomain wesnoth-lib
+###
+### Definition of a spinner widget.
```

Comment:
While a lot of the existing files also have minimal documentation, please improve the docs, rather than just repeating the filename and assuming that people know what a "spinner" is.

Reply:
Updated.